docs/conf.py

```diff
@@ -57,3 +59,10 @@
         (None,),
     ),
 }
+
+# -- Options for sphinx-argparse ------------------------------------------------------
+
+# Limit the width of usage messages. argparse defaults to the terminal width,
+# but we don't want different output depending on the terminal width where the
+# docs were built.
+os.environ["COLUMNS"] = "88"
```

Comment:
Much better, thank you for this change it is indeed much clearer this way 👍🏼

Reply:
I _think_ the last push to the docs was done manually by me, in a fullscreen terminal on a widescreen monitor, and I think that's why the usage summaries are currently a one-line mess with a scrollbar. I don't think we would have gotten a scrollbar had we let CI do an automated publish, because I don't think it would have wound up with the idea that the screen is so wide.

But also to be fair, I had no idea my environment was bleeding into the generated docs like this, so yeah, it's definitely much wiser to hardcode a sane value instead.